  * symbols/group: add ScrollLock as a group switcher key.
    Thanks Andreas Kroschel.  Closes: #343287

Not submitted yet.

Index: xkb-data/symbols/group
===================================================================
--- xkb-data.orig/symbols/group
+++ xkb-data/symbols/group
@@ -346,3 +346,15 @@
     symbols[Group1]= [ space,   ISO_Next_Group ]
     };
 };
+
+// using the group(sclk_toggle) map, pressing the Scroll Lock key
+// toggles groups.
+partial modifier_keys
+xkb_symbols "sclk_toggle" {
+  virtual_modifiers AltGr;
+  key <SCLK> {
+    virtualMods= AltGr,
+    symbols[Group1] = [ ISO_Next_Group ]
+    };
+};
+
Index: xkb-data/rules/base.o_s.part
===================================================================
--- xkb-data.orig/rules/base.o_s.part
+++ xkb-data/rules/base.o_s.part
@@ -37,6 +37,7 @@
   grp:lctrl_toggle	=	+group(lctrl_toggle)
   grp:rctrl_toggle	=	+group(rctrl_toggle)
   grp:lalt_toggle	=	+group(lalt_toggle)
+  grp:sclk_toggle	=	+group(sclk_toggle)
   lv3:switch		=	+level3(switch)
   lv3:ralt_switch	=	+level3(ralt_switch)
   lv3:ralt_switch_multikey	=	+level3(ralt_switch_multikey)
Index: xkb-data/rules/base.xml.in
===================================================================
--- xkb-data.orig/rules/base.xml.in
+++ xkb-data/rules/base.xml.in
@@ -4287,6 +4287,12 @@
           <_description>Right Ctrl</_description>
         </configItem>
       </option>
+      <option>
+        <configItem>
+          <name>grp:sclk_toggle</name>
+          <_description>Scroll Lock</_description>
+        </configItem>
+      </option>
     </group>
     <group allowMultipleSelection="true">
       <!-- The key combination used to choose the 3rd (and 4th, together with Shift)
